### PR TITLE
[stable/karma] add myself to the OWNERS file

### PR DESCRIPTION
**Pull request by prymitive**
As the maintainer of the karma project I'd like to add myself to the OWNERS file to maintain latest version used here.

Signed-off-by: Łukasz Mierzwa l.mierzwa@gmail.com

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed

### CODE REVIEW

File: stable/karma/OWNERS [hunk 1]
```diff
@@ -1,4 +1,6 @@
 approvers:
 - davidkarlsen
+- prymitive
```
Comment (davidkarlsen) [inline stable/karma/OWNERS:3]: I think you need to be in the maintainer section of the Chart.yaml too - see the README

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/karma/OWNERS (hunk 1 above)
Comment (davidkarlsen) [reply stable/karma/OWNERS:3]: and also chart version needs bump to pass lint etc

**Review by davidkarlsen — APPROVED** [review]

File: stable/karma/OWNERS (hunk 1 above)
Comment (prymitive) [reply stable/karma/OWNERS:3]: Yeah, thanks. I missed that part and went straight for `OWNERS`, plus chart version needs a bump too

**Comment by davidkarlsen** [comment]
@prymitive thanks - check your inbox (or github) for an invite to the repo

**Comment by prymitive** [comment]
Thanks, got (and accepted) the invite

**Pull request merged**